tests/fortex/health/processors/scispacy_processor_test.py

```diff
@@ -83,8 +83,8 @@ def test_ScispaCy_Hypomym_processor(self):
         pack = self.nlp.process(document)
         expected_value = {
             "hyponym_link": "such_as",
-            "general": "Keystone plant species",
-            "specific": "fig trees",
+            "specific": "Keystone plant species",
```

Comment:
Shouldn't it be the other way around, like it was before these changes?

Reply:
general is hypernym and specific is hyponym
![image](https://user-images.githubusercontent.com/16308156/212269147-8dff8ad5-ed80-4072-a432-c09c08297c55.png)

Reply:
That's fine, however, what I meant to say was "Keystone plant species" is the general concept while "fig trees" is the specific concept and not the other way around. You can refer https://pythonlang.dev/repo/allenai-scispacy/ to check the ordering on concepts in the returned tuple in Hearst patterns. item[1] should be the general concept and item[2] should be the specific concept, and we should label accordingly. Currently, we are doing the exact opposite I think. Let me know if I have misunderstood.